src/features/Applet/Popups/DuplicatePopups/DuplicatePopups.tsx

```diff
@@ -126,20 +127,33 @@ export const DuplicatePopups = () => {
           </form>
         </StyledModalWrapper>
       </Modal>
-      <EnterAppletPasswordPopup
-        popupVisible={passwordModalVisible}
-        setPopupVisible={setPasswordModalVisible}
-        submitCallback={submitCallback}
-      />
-      <Modal
-        open={successModalVisible}
-        onClose={successModalClose}
-        title={t('appletDuplication')}
-        onSubmit={successModalClose}
-        buttonText={t('dismiss')}
-      >
-        <StyledModalWrapper>{t('successDuplication')}</StyledModalWrapper>
-      </Modal>
+      {passwordModalVisible && (
+        <AppletPasswordPopup
+          onClose={passwordModalClose}
+          popupType={AppletPasswordPopupType.CREATE}
+          popupVisible={passwordModalVisible}
+          submitCallback={submitCallback}
+        />
+      )}
+      {successModalVisible && (
+        <Modal
+          open={successModalVisible}
+          onClose={successModalClose}
+          title={t('appletDuplication')}
+          onSubmit={successModalClose}
+          buttonText={t('OK')}
```

Comment:
Is it fits with the uppercase 'OK', because, in the file app-en.json, there is 'ok'?